tests/foreman/cli/test_host.py

```diff
@@ -1782,6 +1786,7 @@ def test_positive_erratum_applicability(
 
 @pytest.mark.cli_katello_host_tools
 @pytest.mark.tier3
+@pytest.mark.parametrize('setup_custom_repo', [{'custom_repo': 'yum_3'}], ids=[''], indirect=True)
```

Comment:
The PRT run failed to collect tests because setting an `id` for a parameter doesn't make it disappear. 

Test collection for this function looks like:
```
  <Function test_positive_apply_security_erratum[rhel6-]>
    <Function test_positive_apply_security_erratum[rhel7-]>
    <Function test_positive_apply_security_erratum[rhel7_fips-]>
    <Function test_positive_apply_security_erratum[rhel8-]>
    <Function test_positive_apply_security_erratum[rhel8_fips-]>
    <Function test_positive_apply_security_erratum[rhel9-]>
```

Reply:
PRT collected, but I don't think setting this ID to an empty string is a good pattern - when looking at test results you can tell there was another parameter, but its missing, leading to confusion.  Why not provide IDs for the two parameters?

Reply:
Added "yum3" as id, and now the test collection looks like this:
![image](https://user-images.githubusercontent.com/43444182/175028337-b15cce38-1520-45f6-991f-fc0192bb518b.png)

if you have any better ideas for ID let me know!